### PR TITLE
[docs][trivy] Add wide description about trivy storageClass usage

### PR DESCRIPTION
**Pull request by haosb**
## Description
Update trivy documentation with wide description about storageClass.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: trivy
type: fix
summary: Add wide description about trivy storageClass usage
impact_level: default
```

### CODE REVIEW

File: ee/modules/500-operator-trivy/openapi/doc-ru-config-values.yaml [hunk 1]
```diff
@@ -2,9 +2,12 @@ type: object
 properties:
   storageClass:
     description: |
-      Имя StorageClass.
+      Имя storageClass'а, который будет использоваться.
 
-      `false` — принудительное использование `emptyDir`. После установки необходимо вручную удалить старый PVC и перезапустить под.
+      * Используется или [глобальный StorageClass](../../deckhouse-configure-global.html#parameters-storageclass), или `global.discovery.defaultStorageClass`, а если и их нет, данные сохраняются в emptyDir.
```
Comment (mortelumina98) [inline ee/modules/500-operator-trivy/openapi/doc-ru-config-values.yaml:7]:
```suggestion
      
      * Используется [глобальный StorageClass](../../deckhouse-configure-global.html#parameters-storageclass), или `global.discovery.defaultStorageClass`. Если ни один из них не задан, данные сохраняются в emptyDir.
      
```

File: ee/modules/500-operator-trivy/openapi/doc-ru-config-values.yaml [hunk 2]
```diff
@@ -2,9 +2,12 @@ type: object
...
+      * `global.discovery.defaultStorageClass` применяется только при включении модуля, смена StorageClass по умолчанию в кластере не приведет к перезаказу диска.
+      * **ОСТОРОЖНО!** При указании этой опции в значение, отличное от текущего (из существующей PVC), диск будет перезаказан, а все данные удалены.
```
Comment (mortelumina98) [inline ee/modules/500-operator-trivy/openapi/doc-ru-config-values.yaml:9]:
```suggestion
      * **Внимание.** Если указать значение, отличное от текущего (используемого в существующей PVC), диск будет перезаказан, и все данные удалятся.
```

File: ee/modules/500-operator-trivy/openapi/config-values.yaml [hunk 3]
```diff
@@ -6,7 +6,14 @@ properties:
     description: |
       The name of the StorageClass to use.
 
-      `false` — forces the `emptyDir` usage. Manually delete the old PVC and restart Pod, after setting the parameter.
+      If omitted, the StorageClass of the existing PVC is used. If there is no PVC yet, either the global [StorageClass](../../deckhouse-configure-global.html#parameters-storageclass) or `global.discovery.defaultStorageClass` is used, and if those are undefined, the emptyDir volume is used to store the data.
+
+      `global.discovery.defaultStorageClass` is applied during module activation, changing default StorageClass in cluster won't result in disk re-provisioning.
+
+      **CAUTION!** Setting this value to one that differs from the current one (in the existing PVC) will result in disk re-provisioning and data loss.
```
Comment (mortelumina98) [inline ee/modules/500-operator-trivy/openapi/config-values.yaml:13]:
```suggestion
      **Warning.** Setting this value to one that differs from the current one (in the existing PVC) will result in disk re-provisioning and data loss.
```

File: ee/modules/500-operator-trivy/openapi/doc-ru-config-values.yaml [hunk 4]
```diff
@@ -2,9 +2,12 @@ type: object
 properties:
   storageClass:
     description: |
-      Имя StorageClass.
+      Имя storageClass'а, который будет использоваться.
 
-      `false` — принудительное использование `emptyDir`. После установки необходимо вручную удалить старый PVC и перезапустить под.
+      * Используется или [глобальный StorageClass](../../deckhouse-configure-global.html#parameters-storageclass), или `global.discovery.defaultStorageClass`, а если и их нет, данные сохраняются в emptyDir.
+      * `global.discovery.defaultStorageClass` применяется только при включении модуля, смена StorageClass по умолчанию в кластере не приведет к перезаказу диска.
```
Comment (mortelumina98) [inline ee/modules/500-operator-trivy/openapi/doc-ru-config-values.yaml:8]:
```suggestion
      * `global.discovery.defaultStorageClass` применяется только при включении модуля, смена StorageClass по умолчанию в кластере не приведет к перезаказу диска.
      
```

File: ee/modules/500-operator-trivy/openapi/doc-ru-config-values.yaml [hunk 5]
```diff
@@ -2,9 +2,12 @@ type: object
...
+      * **ОСТОРОЖНО!** При указании этой опции в значение, отличное от текущего (из существующей PVC), диск будет перезаказан, а все данные удалены.
+      * Если указать `false`, будет форсироваться использование emptyDir'а.
```
Comment (mortelumina98) [inline ee/modules/500-operator-trivy/openapi/doc-ru-config-values.yaml:10]:
```suggestion
      
      * Если указать `false`, будет форсироваться использование emptyDir'а.
      
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10285](https://github.com/deckhouse/deckhouse/pull/10285) to the branch [release-1.65](https://github.com/deckhouse/deckhouse/tree/release-1.65) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10286](https://github.com/deckhouse/deckhouse/pull/10286) to the branch [release-1.64](https://github.com/deckhouse/deckhouse/tree/release-1.64) successful!

**Pull request merged**